api/app/src/external/commonwell/document/shared.ts

```diff
@@ -3,9 +3,11 @@ import { contentType, extension } from "mime-types";
 import { Patient } from "../../../models/medical/patient";
 import { makePatientOID } from "../../../shared/oid";
 
-export type DocumentWithFilename = Document & {
-  fileName: string;
-  raw?: unknown;
```

Comment:
`raw` was introduced bc we used to store this on Postgres... now its not useful anymore, even Sentry can't make good use of it:

<img width="177" alt="image" src="https://github.com/metriport/metriport/assets/2132564/34045594-da26-4d43-a656-56b4662b48d2">